modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/Pages/create.js

```diff
@@ -171,6 +166,11 @@ $(function () {
         });
     }
 
+    $('#GeneratedWidgetText').click(function () {
```

Comment:
After making it 'change' instead of 'click':

```suggestion
    $('#GeneratedWidgetText').on('change', function () {
```